RELEASE_GUIDE.md

```diff
@@ -117,9 +117,15 @@ considering what effects this may have:
   point.
 
 - After all of the prebuilt binaries are present on the GitHub releases page,
-  publish to npm: change to the `npm-package` directory and run `npm publish`.
-  It's a good idea to check that the package can be installed from npm at this
-  point.
+  publish to npm: change to the `npm-package` directory and do the following:
+    - if making a pre-release (e.g. `v0.15.0-alpha-05`)
+        - run `npm publish --tag next`
+        - verify that the prerelease can be installed via `npm i purescript@next`
+    - if making a normal release (e.g. `v0.15.0`)
+        - run `npm publish --tag next`
+        - run `npm publish`
```

Comment:
I suspect this won't work; I think `npm` prevents you from ever publishing the same version twice. You probably want

```suggestion
        - run `npm publish`
        - run `npm dist-tag add purescript@x.y.z next`
```

(maybe you can get away with replacing `x.y.z` with `latest`, I don't know)

Reply:
I'm going to create a small npm package and see what happens when I run `npm publish --tag next` multiple times.

Reply:
You're correct. Running `npm publish --tag next` then `npm publish` produces this error:
```
npm ERR! code E403
npm ERR! 403 403 Forbidden - PUT https://registry.npmjs.org/trashme-purs-prerelease-test - You cannot publish over the previously published versions: 1.0.1-0.
npm ERR! 403 In most cases, you or one of your dependencies are requesting
npm ERR! 403 a package version that is forbidden by your security policy, or
npm ERR! 403 on a server you do not have access to.
```

Reply:
So, after playing around with a toy package, here's what I've discovered.

First, running `npm publish` and then `npm dist-tag add package@v1.2.3 next` works (as Ryan stated above).

But because this is still manual, second, I explored how to fix a bad publish. For example, let's assume I ran
```
# wrong version: I meant `v1.2.0`
npm dist-tag add package@v1.1.0 next
# let's just fix it by updating `next` right?
npm dist-tag add package@v1.2.0 next
```
Let's assume the following:
- a package `bar`'s `next` tag refers to `v1.0.0`
- a package `foo` installed a package `bar` via `npm i bar@next`, so that `foo` depends on `bar@v1.0.0`.
- `npm version minor && npm publish --tag next` (so `bar@next` now points to `v1.1.0`)
- locally in `foo`, I run `npm i bar@next`, `v1.0.0` is installed, which is incorrect as it should be `v1.1.0` now.
- locally in `foo`, I nuke `node_modules`, `package-lock.json`, and remove `bar` from my dependencies. When I run `npm i bar@next`, `v1.0.0` is still installed.
- locally in `foo`, running `npm cache clear --force` followed by `npm i bar@next` now installs `bar@v1.1.0` in `foo`.

Does the "`next` tag sometimes points to the real `next` tag" issue above only happen locally because I'm publishing it on the same computer? Or would the same thing happen on other computers (e.g. in CI)?